### PR TITLE
PXB-2603 Release Notes for 2.4.24 (2.4)

### PR DESCRIPTION
**Pull request by patrickbirch**
Modified files:
conf.py
release-notes
Added file:
2.4.24

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot tolerate
+long periods of downtime. Offered free as an open source solution, it drives down backup
+costs while providing unique features for MySQL backups.
+
+Improvements
+================================================================================
+
+* :jirabug:`PXB-2477`: The xbcloud Binary should retry on error and utilize incremental backoff
+* :jirabug:`PXB-2580`: xbcloud: a chunk-upload on SSL connect error to Amazon S3 was not retried.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst]: Please add the credits as per 8.0 RN

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-762`: Document the recovery process with relay-log-info-repository=TABLE
+* :jirabug:`PXB-1504`: FIND_GCRYPT macro is broken
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst]: Same here.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst]: Added.

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.24.rst]: Added.

**Review by altmannmarcelo — APPROVED** [review]

**Review by mchawla16 — APPROVED** [review]

**Pull request merged**